engine/src/flutter/lib/ui/window.dart

```diff
@@ -913,6 +913,47 @@ class SingletonFlutterWindow extends FlutterView {
   void setIsolateDebugName(String name) => PlatformDispatcher.instance.setIsolateDebugName(name);
 }
 
+/// The system-reported typography settings if any.
+//
+// When changes are made to this class, the equivalent APIs in each of the
+// embedders *must* be updated.
+final class TypographySettings {
+  /// Creates a new [TypographySettings].
+  const TypographySettings({
+    this.lineHeight,
+    this.letterSpacing,
+    this.wordSpacing,
+    this.paragraphSpacing,
+  });
+
+  /// The height of the text, as a multiple of the font size.
+  final double? lineHeight;
```

Comment:
Should this be `lineHeightFactor` instead? It seems like a more appropriate name. 

In CSS `line-height` can accept:
- Default: normal
- `Normal` keyword
- Unitless, relative to font size, for example `2`.
- <length>, number followed by units `10px`
- <percentage>, number followed by % sign `10%`, relative to font size

but there seems to be a strong preference for unitless numbers, i.e. numbers relative to font size.

https://developer.mozilla.org/en-US/docs/Web/CSS/line-height#prefer_unitless_numbers_for_line-height_values

In the extension and bookmarklet I tried both are using unitless values for their CSS.

Reply:
IMO it doesn't make sense to set line height to an absolute number for a11y purposes. Percentage makes sense but it's basically the same as unitless, i.e. `150% == 1.5`.